docs/plugins/default/highlight.md

```diff
@@ -0,0 +1,26 @@
+# Highlight plugin
+
+Base plugin enabled by default. Highlights code in subject of commit. 
+
+Default options:
+
+```YAML
+
+camelCase: true
+masks: []
+
+```
+
+## camelCase
+
+Default: `true`
+
+In the case of false won't highlight camel case code.
```

Comment:
Highlights a word or abbreviation written in [camelCase](https://en.wikipedia.org/wiki/Camel_case)